production/loki-mixin/dashboards/dashboard-utils.libsonnet

```diff
@@ -47,15 +47,20 @@ local utils = import 'mixin-utils/utils.libsonnet';
           d.addMultiTemplate('cluster', 'loki_build_info', 'cluster')
           .addMultiTemplate('namespace', 'loki_build_info', 'namespace')
         else
-          d.addTemplate('cluster', 'loki_build_info', 'cluster')
```

Comment:
Can you help me understand why we should remove the `cluster` label here?

Reply:
Sorry about that! This shouldnt have been removed.